docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-creating-a-case-proc.adoc

```diff
@@ -1,3 +1,32 @@
 [id='case-management-creating-a-case-proc']
 = Creating a case in {PRODUCT}
```

Comment:
I would remove {PRODUCT} from this chapter title. Just imho :) No harm either way.